gocourse07/birdaviary/go.mod

```
@@ -0,0 +1,4 @@
+//module github.com/sarff/prjctr-golang_beginning/gocourse07/birdaviary
+module birdaviary
```

Ось так же має працювати:
```suggestion
module github.com/sarff/prjctr-golang_beginning/gocourse07/birdaviary
```